packages/utils/src/loader/svg-loader.ts

```diff
@@ -7,6 +7,13 @@ import { CDNLoader } from './cdn-loader.js';
  */
 const isUrl = (str: string): boolean => (/^(https?:\/{2}|\.?\/)/i).test(str);
 
+/**
+ * Checks a string to see if it's a base64 URL
+ * @param str String to test
+ * @returns is Base64
+ */
+const isBase64svg = (str: string): boolean => (/^(data:image\/[svg,svg+xml]+;base64,[a-zA-Z0-9,+,/]+={0,2})/i).test(str);
```

Comment:
Very terse 👍

Reply:
Yes, I like it for real ❤️

Reply:
Thanks for this regex101 topic, there really save my day. hehe
https://regex101.com/library/XAQEkg?filterFlavors=javascript&filterFlavors=golang&orderBy=MOST_RECENT&search=

Reply:
By the way, this regular expression cause the 2 code smells on this PR 😅